packaging/systemd/microshift-containerized.service

```diff
@@ -13,9 +13,9 @@ RequiresMountsFor=%t/containers
 Environment=PODMAN_SYSTEMD_UNIT=%n
 Restart=on-failure
 TimeoutStopSec=70
+ExecStartPre=/bin/bash /usr/bin/microshift-containerized # not autogenerated
 ExecStartPre=/bin/rm -f %t/%n.ctr-id
-ExecStartPre=/usr/bin/mkdir -p /var/hpvolumes
-ExecStart=/usr/bin/podman run --cidfile=%t/%n.ctr-id --cgroups=no-conmon --rm --replace --sdnotify=container --label io.containers.autoupdate=registry --network=host --privileged -d --name microshift -v /var/run:/var/run -v /sys:/sys:ro -v /var/lib:/var/lib:rw,rshared -v /var/hpvolumes:/var/hpvolumes -v /lib/modules:/lib/modules -v /etc:/etc -v /run/containers:/run/containers -v /var/log:/var/log -e KUBECONFIG=/var/lib/microshift/resources/kubeadmin/kubeconfig quay.io/microshift/microshift:latest
+ExecStart=/usr/bin/podman run --cidfile=%t/%n.ctr-id --cgroups=no-conmon --rm --replace --sdnotify=container --label io.containers.autoupdate=registry --network=host --privileged -d --name microshift -v /var/run:/var/run -v /sys:/sys:ro -v /var/lib:/var/lib:rw,rshared -v /var/hpvolumes:/var/hpvolumes -v /lib/modules:/lib/modules -v /etc:/etc -v /run/containers:/run/containers -v /var/log:/var/log -e KUBECONFIG=/var/lib/microshift/resources/kubeadmin/kubeconfig -v microshift-persistent-data:/var/hpvolumes quay.io/microshift/microshift:latest
```

Comment:
This command line is attempting to mount `/var/hpvolumes` twice. It has both:

```
-v /var/hpvolumes:/var/hpvolumes
```

And:

```
-v microshift-persistent-data:/var/hpvolumes
```